scripts/globals/events/harvest_festival.lua

```diff
@@ -0,0 +1,318 @@
+        { name = "Trick_Phantom",  costume = 0 },
+        { name = "Trick_Ghost",    costume = 0 },
+        { name = "Trick_Wight",    costume = 0 },
+        { name = "Trick_Ghast",    costume = 0 },
+        { name = "Trick_Bones",    costume = 0 },
+        { name = "Trick_Skeleton", costume = 0 },
+    },
+
+    [xi.zone.BASTOK_MINES] =
+    {
+        { name = "Faustin",      costume = 365 }, -- Hound
+        { name = "Mille",        costume = 644 }, -- Quadav
+        { name = "Aulavia",      costume = 368 }, -- Ghost
+        { name = "Proud_Beard",  costume = 544 }, -- Shade
+        { name = "Emaliveulaux", costume = 564 }, -- Skeleton
+    },
+
+    [xi.zone.BASTOK_MARKETS] =
+    {
+        { name = "Olwyn",          costume = 368 }, -- Ghost
+        { name = "Trick_Spirit",   costume = 0 },
+        { name = "Trick_Specter",  costume = 0 },
+        { name = "Trick_Shadow",   costume = 0 },
+        { name = "Trick_Shade",    costume = 0 },
+        { name = "Trick_Phantom",  costume = 0 },
+        { name = "Trick_Ghost",    costume = 0 },
+        { name = "Trick_Wight",    costume = 0 },
+        { name = "Trick_Ghast",    costume = 0 },
+        { name = "Trick_Bones",    costume = 0 },
+        { name = "Trick_Skeleton", costume = 0 },
+    },
+}
+
+local costumeList =
+{
+    math.random(531, 534), -- Stalker
+    math.random(580, 607), -- Yagudo
+    math.random(644, 671), -- Quadav
+    math.random(535, 538), -- Shade
+    math.random(707, 711), -- Gigas
+    math.random(740, 756), -- Demon
+    math.random(612, 639), -- Orc
+    math.random(484, 511), -- Goblin
+    368,                   -- Ghost
+    365,                   -- Hound
+    564,                   -- Skeleton
+}
+
+local treatsTable2005 =
+{
+    4510, -- Acorn Cookie
+    5646, -- Bloody Chocolate
+    4496, -- Bubble Chocolate
+    4397, -- Cinna-cookie
+    4394, -- Ginger Cookie
+    4495, -- Goblin Chocolate
+    4413, -- Apple Pie
+    4488, -- Jack-o'-Pie
+    4421, -- Melon Pie
+    4563, -- Pamama Tart
+    4446, -- Pumpkin Pie
+    4414, -- Rolanberry Pie
+    4406, -- Baked Apple
+    5729, -- Bavarois
+    5745, -- Cherry Bavarois
+    5653, -- Cherry Muffin
+    5655, -- Coffee Muffin
+    5718, -- Cream Puff
+    5144, -- Crimson Jelly
+    5681, -- Cupid Chocolate
+    5672, -- Dried Berry
+    5567, -- Dried Date
+    4556, -- Icecap Rolanberry
+    5614, -- Konigskuchen
+    5230, -- Love Chocolate
+    4502, -- Marron Glace
+    4393, -- Orange Kuchen
+    5147, -- Snoll Gelato
+    4270, -- Sweet Rice Cake
+    5645, -- Witch Nougat
+    5552, -- Black Pudding  --safe
+    5550, -- Buche au Chocolat -- safe @ 43 items
+    5616, -- Lebkuchen House --breaks
+    5633, -- Chocolate Cake
+    5542, -- Gateau aux Fraises
+    5572, -- Irmik Helvasi
+    5625, -- Maple Cake
+    5559, -- Mille Feuille
+    5557, -- Mont Blanc
+    5629, -- Orange Cake
+    5631, -- Pumpkin Cake
+    5577, -- Sutlac
+    5627, -- Yogurt Cake
```

Comment:
These could be added to items.lua

Reply:
What do these comments mean?
```
    5552, -- Black Pudding  --safe
    5550, -- Buche au Chocolat -- safe @ 43 items
    5616, -- Lebkuchen House --breaks
    ```

Reply:
Redundant code from what was pre-existing

Reply:
Items converted!